### DISCUSSION TITLE
Just wanted to clear some confusion, does this work on Intel Macs?

### DISCUSSION THREAD

**Discussion by V-R-Technology** [Whisky-App/Whisky#70] [Q&A] [answered]
Asking for this program and when it is officially released.

**Comment by evanpaul14** [comment, marked as answer]
It does (I haven’t seen anyone test it with real games tho) but if you have an Intel Mac it would be simpler to use boot camp and install windows. You’ll get better performance.

**Reply by V-R-Technology** [reply]
I agree, however I use an eGPU and it’s a nuisance to get that working on boot camp, and I have a low SSD capacity.
